gnomad_qc/v4/resources/annotations.py

```diff
@@ -407,6 +413,32 @@ def get_vrs(
     )
 
 
+def hgdp_tgp_updated_callstats(
+    test: bool = False, subset: str = "final"
+) -> VersionedTableResource:
+    """
+    Get the HGDP + 1KG/TGP subset release MatrixTableResource.
+
+    :param test: If True, will return the annotation resource for testing purposes.
+    :param subset: The subset of the HGDP + 1KG/TGP release to return,
+       must be "added", "subtracted", "pop_diff", or "final"
+    :return: MatrixTableResource for specified subset.
+    """
+    if subset not in ["added", "subtracted", "pop_diff", "final"]:
+        raise ValueError(
+            "Operation must be one of 'added', 'subtracted', 'pop_diff', or 'final'"
+        )
```

Comment:
```suggestion
    if subset not in ["added", "subtracted", "pop_diff", "final"]
        raise ValueError(
            "Operation must be one of 'added', 'subtracted', 'pop_diff', or 'join'"
        )
```
The `final` has to be changed to using `get_freq`, right?

Reply:
yup, thanks!